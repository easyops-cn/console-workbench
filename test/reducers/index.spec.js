/* eslint-disable import/first */
jest.mock('../../app/storage');

import rootReducers from '../../app/reducers';

describe('reducers', () => {
  describe('root', () => {
    it('should handle initial state', () => {
      expect(
        rootReducers({
          location: '/'
        })(undefined, {})
      ).toMatchSnapshot();
    });
  });
});
