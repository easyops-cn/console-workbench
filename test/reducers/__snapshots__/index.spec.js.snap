// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers root should handle initial state 1`] = `
Object {
  "jobs": Object {
    "active": null,
    "entities": Object {},
    "ids": Array [],
    "outputs": Object {},
  },
  "router": Object {
    "action": undefined,
    "location": "/",
  },
}
`;
